index.html

```diff
@@ -26,36 +26,191 @@ <h4>Rabaya Rabu</h4>
+                <button type="button">See project</button>
+            </article>
+            <article class="card">
+                <h5>Profesional Art Printing Data</h5>
+                <p>A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required. Has been the industry's standard.
+                </p>
+                <ul>
+                    <li class="list1">HTML</li>
+                    <li class="list2">Bootstrap</li>
+                    <li class="list3">Ruby</li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+            <article class="card">
+                <h5>Profesional Art Printing Data</h5>
+                <p>A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required. Has been the industry's standard.
+                </p>
+                <ul>
+                    <li class="list1">HTML</li>
+                    <li class="list2">Bootstrap</li>
+                    <li class="list3">Ruby</li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+            <article class="card">
+                <h5>Profesional Art Printing Data</h5>
+                <p>A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required. Has been the industry's standard.
+                </p>
+                <ul>
+                    <li class="list1">HTML</li>
+                    <li class="list2">Bootstrap</li>
+                    <li class="list3">Ruby</li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+            <article class="card">
+                <h5>Profesional Art Printing Data</h5>
+                <p>A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required. Has been the industry's standard.
+                </p>
+                <ul>
+                    <li class="list1">HTML</li>
+                    <li class="list2">Bootstrap</li>
+                    <li class="list3">Ruby</li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+            <article class="card">
+                <h5>Profesional Art Printing Data</h5>
+                <p>A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required. Has been the industry's standard.
+                </p>
+                <ul>
+                    <li class="list1">HTML</li>
+                    <li class="list2">Bootstrap</li>
+                    <li class="list3">Ruby</li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+        </section>
+        <!-- about section -->
+        <section class="about">
+            <h1>About me</h1>
```

Comment:
Kindly update this element to use another `<hx>` element apart from `<h1>`🙏. This is for semantic reasons, as a good webpage is typically supposed to have one `<h1>` tag which represents the main title